### PR TITLE
build: Update node image

### PR DESCRIPTION
**Pull request by ynnoj**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 defaults: &defaults
   docker:
-    - image: circleci/node:4-browsers
+    - image: circleci/node:6-browsers
```
Comment (marcqualie) [inline .circleci/config.yml]: Any reason we can't upgrade to `node:8.11-browsers` (LTS version)?

Comment (ynnoj) [reply .circleci/config.yml]: @marcqualie As this is an addon, the consuming application could be running a whole array of Ember.js (as subsequently Node) versions. `node:6` is currently inline with the Ember.js LTS versioning.

https://github.com/ember-cli/ember-cli/blob/master/docs/node-support.md

Comment (ynnoj) [reply .circleci/config.yml]: Not that this fixed my issue anyway 😓

Comment (ynnoj) [reply .circleci/config.yml]: @marcqualie Turns out `semantic-release` needs Node 8 anyway 😂 

https://github.com/semantic-release/semantic-release/blob/caribou/docs/support/node-version.md#node-version-requirement

**Pull request merged**